### ISSUE TITLE
Replace incomplete gst-opera folder + gstreamer.dll binary blob with actual src

### ISSUE THREAD

**Issue by PrestoXen** [PrestoXen/openopera-issues#15]
So for Windows users we're stuck relying on the pre-built gstreamer.dll included in the source tree.. which means no VP9 hacks for us :(

They used to host "all" the code for it under opensource.opera.com, but they obviously don't care about that server so getting code from it was a hassle...

Eventually after some persuasion the server complied, and you can download the full code + git repos here: http://www65.zippyshare.com/v/4YKByv4p/file.html

Unfortunately in my initial test a gstreamer.dll built from that code would break all video playback, one problem after another...

I explained in the thread how my test was pretty flawed though, and outlined some reasons it might have failed:

* Building with VS2015, testing on a VS2015 build, while VS2015 builds are still unstable atm
* Unsure if the gstreamer I built was 32 or 64, was testing with x64 opera though
* libvpx used to build gst was 1.3, Opera's was probably older
* There's small parts of gst-opera included in the leaked source, which maybe we have to merge with this repo
* or worst case: this public gst-opera repo is missing some opera-specific changes

So yeah, pretty flawed way of testing :) (to be fair it was something like 4AM..)

Going to try again later with a proper test, wish me luck.

**Comment by PrestoXen** [comment]
And it turns out x64 builds won't work with this source.. great, like I said, one problem after another :)

**Comment by PrestoXen** [comment]
Alright got around the x64 build issue, and it also seems to work fine with Opera too :) No longer have to rely on the gstreamer.dll binary blob 👍 

You can download the repos here, they should build/run fine under VS2010/VS2015 unless I messed something up: http://www22.zippyshare.com/v/Wp1PrPRW/file.html

Make sure to replace the libvpx in there with the one your Opera build is using.

**Comment by Zero3K** [comment]
I think that you should upload the working source code to Github.

**Comment by PrestoXen** [comment]
Yeah I've been meaning to, I think I'll do it like rikaunite's repos at https://github.com/rikaunite/gst-opera https://github.com/rikaunite/gst-opera_glib https://github.com/rikaunite/gst-opera_gst-plugins-good etc.

Can probably add it as a submodule to the main Opera repo then too, so we'd have a full source tree instead of having binary blobs like gstreamer.dll (probably only really worth it if we can find a git host that won't shut down the Opera repo though)

**Comment by Zero3K** [comment]
Any news regarding this issue?

**Comment by Zero3K** [comment]
Any news regarding this issue?
